common/edqs/src/main/java/org/thingsboard/server/edqs/processor/EdqsProcessor.java

```diff
@@ -277,6 +277,7 @@ public void destroy() throws InterruptedException {
         eventConsumer.awaitStop();
         responseTemplate.stop();
         stateService.stop();
+        versionsStore.shutdown();
```

Comment:
VersionsStore is also used KafkaEdqsStateService

Reply:
good catch, thanks. fixed